docs/UserGuideCS2101.md

```diff
@@ -0,0 +1,424 @@
+User Guide
+=======
+SETA is a **desktop application for CS2103T Teaching Assistants** to track their students’ and tutorials’ details, and
+questions asked by students. SETA is optimized for use via a Command Line Interface (CLI) while still having the
+benefits of a Graphical User Interface (GUI). If you can type fast, SETA enables you to track your students, manage your
+tutorials and note down questions more effectively than traditional GUI apps.
+
+The purpose of this guide is to inform you what features are available in SETA, and how to use them to solve your 
+needs as a CS2103T TA.
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: What is a Command Line Interface (CLI)?**<br>
```

Comment:
May want to include what icons mean here if we plan to use icons at the start of the user guide :)

Reply:
Maybe I can leave it to the rest? I'm still not sure how we can improve the reader friendliness.
There's also the navigation issue that I have not resolved yet.